apps/rule-manager/app/service/RuleManager.scala

```diff
@@ -267,9 +279,10 @@ object RuleManager extends Loggable {
     }
   }
 
-  def archiveRule(
+  def unpublishRule(
       id: Int,
-      user: String
+      user: String,
+      bucketRuleResource: BucketRuleResource
   ): Either[Exception, (Option[DbRuleDraft], Option[DbRuleLive])] = {
```

Comment:
There's an argument to be made that we could return `AllRuleData` here, in line with `publishRule`.

Reply:
Non-blocking: IIUC we will probably need to do that in future if we'd like to improve our rule history by providing the end date of the published rule, as well as the start date.

(The end date will be the new information provided by `AllRuleData`.)

Could include that behaviour in this PR, although there's no UI I'm aware of that will surface the update at the moment.